qa/die-with-dignity/src/test/java/org/elasticsearch/qa/die_with_dignity/DieWithDignityIT.java

```diff
@@ -60,21 +60,19 @@ public void testDieWithDignity() throws Exception {
 
         // parse the logs and ensure that Elasticsearch died with the expected cause
         final List<String> lines = Files.readAllLines(PathUtils.get(System.getProperty("log")));
-
         final Iterator<String> it = lines.iterator();
 
         boolean fatalError = false;
         boolean fatalErrorInThreadExiting = false;
         try {
             while (it.hasNext() && (fatalError == false || fatalErrorInThreadExiting == false)) {
                 final String line = it.next();
-                if (line.matches(".*ERROR.*o\\.e\\.ExceptionsHelper.*integTest-0.*fatal error.*")) {
+                if( containsAll(line,".*ERROR.*", ".*ExceptionsHelper.*", ".*integTest-0.*", ".*fatal error.*")) {
```

Comment:
Whitespace:

```suggestion
                if (containsAll(line,".*ERROR.*", ".*ExceptionsHelper.*", ".*integTest-0.*", ".*fatal error.*")) {
```

...or opt into formatting 😅

Reply:
will opt in modules related to logging to formatting